packages/terra-demographics-banner/tests/jest/__snapshots__/DemographicsBannerDisplay.test.jsx.snap

```diff
@@ -3,7 +3,7 @@
 exports[`renders a banner that has basic information 1`] = `
 <ResponsiveElement
   defaultElement={
-    <Component
+    <Unknown
```

Comment:
What is "Unknown"?

Reply:
Not entirely sure, I tested this with shallow, mount, and render and all output the `string` unknown here. Looks like jest has some logic to determine the type of component for react elements / components which are passed in as props [here](https://github.com/facebook/jest/blob/master/packages/pretty-format/src/plugins/react_element.js#L34-L42).

With how [_LargeDemographicsBannerDisplay](https://github.com/cerner/terra-core/blob/master/packages/terra-demographics-banner/src/_LargeDemographicsBannerDisplay.jsx#L10) and [_SmallDemographicsBannerDisplay](https://github.com/cerner/terra-core/blob/master/packages/terra-demographics-banner/src/_SmallDemographicsBannerDisplay.jsx#L10) are exported, it seems jest is unable to determine what type they are and falls back to using the string `Unknown` where before it would use the string `Component`.